proposed/http-middleware/middleware-meta.md

```diff
@@ -123,7 +130,7 @@ Middleware taking this approach generally has the following commonalities:
   the request for processing.
 * The middleware is passed 2 arguments during invocation:
   1. An object that represents an HTTP request.
-  2. A delegate that receives the request to dispatch next middleware in the pipeline.
+  2. A handler that receives the request to dispatch next middleware in the pipeline.
```

Comment:
How about:

> A handler to which the middleware can delegate the responsibility of producing a response.

I think having the term "delegate" in the sentence helps connecting the concepts?